knowledge-base/how-to-create-multiselect-checkbox-parameter-editor.md

```diff
@@ -0,0 +1,107 @@
+
+The following example illustrates how to use the [Kendo CheckBoxGroup](https://docs.telerik.com/kendo-ui/controls/editors/checkboxgroup/overview) widget for a multi parameter value parameter editor which also has available values.
+
+## Solution
+
+1. Add a reference to kendo of version R1 2021 at least
+
+```HTML
+<script src="https://kendo.cdn.telerik.com/2021.1.119/js/kendo.all.min.js"></script>
+```
+2. Create a match and editor functions
+
+```JavaScript
+       
+       function matchFunction(parameter) {
+            return Boolean(parameter.availableValues) && parameter.multivalue;
+        }
+
+        function createCustomEditor(placeholder, options) {
+            let checkboxGroupElement = $(placeholder).html("<ul></ul>");
+            let valueChangedCallback = options.parameterChanged;
+            let parameter;
+
+            function onChange() {
+                let val = checkboxList.value();
+                valueChangedCallback(parameter, val);
+            }
+
+            return {
+                beginEdit: function (param) {
+
+                    parameter = param;
+
+                    $(checkboxGroupElement).kendoCheckBoxGroup({
+                        layout: "vertical",
+                        items: parameter.availableValues.map((param) => ({ ...param, label: param.name })),
+                        change: onChange,
+                        value: parameter.value
+                    });
+
+                    checkboxList = $(checkboxGroupElement).data("kendoCheckBoxGroup");
+                }
+            };
+        }
+```
+3. Edit the **parameterEditors** array:
+
+```JavaScript
+  $(document).ready(function () {
+            $("#reportViewer1")
+                .telerik_ReportViewer({
+                    serviceUrl: "api/reports/",
+                    reportSource: {
+                        report: "Product Line Sales.trdp",
+                        parameters: {}
+                    },
+                    parameterEditors: [{
+                        match: matchFunction,
+                        createEditor: createCustomEditor
+                    }],
+                    viewMode: telerikReportViewer.ViewModes.INTERACTIVE,
+                    scaleMode: telerikReportViewer.ScaleModes.FIT_PAGE,
+                    scale: 1.0,
+                    enableAccessibility: false,
+                    sendEmail: { enabled: true },
+                });
+
+        });
+```
+
+## See Also
+[What's new in Kendo UI for jQuery](https://www.telerik.com/blogs/whats-new-kendo-ui-jquery-r1-2021) 
+
+[How to Create a Custom Parameter Editor](https://docs.telerik.com/reporting/html5-report-viewer-howto-custom-parameter-editor)
```

Comment:
Again, you can use relative path ../reporting/html5-report-viewer-howto-custom-parameter-editor